gcp/rakefiles/test.rake

```diff
@@ -0,0 +1,28 @@
+desc '[TEST] Run Locust swarm against Preferences service in current cluster'
+task :test_preferences => [:set_vars] do
+  # TODO: Once GPII-3250 is resolved, TF_VAR_locust_desired_max_response_time
+  # should be changed back to it's default value of 500
+  sh "#{@exekube_cmd} rake xk[' \
+    xk down live/dev/locust && \
+    echo \"Waiting for K8s to fully terminate Locust resources...\" && \
+    sleep 45 && \
+    TF_VAR_locust_target_host=http://preferences.$TF_VAR_domain_name \
+    TF_VAR_locust_script=preferences.py \
+    TF_VAR_locust_desired_max_response_time=4000 \
+    xk up live/dev/locust',skip_infra,skip_secret_mgmt]"
```

Comment:
So the locust cluster will stay up after running this task, and we'll try to tear down existing one at the beginning of the each test task?

Reply:
Main reason why we destroy Locust module at the beginning and not after tests – if tests fail, execution would be stopped and module destroy code would never run. We have separate `destroy_module[locust]` step for `stg` and `prd`.